src/components/collapsible-section/collapsible-section.scss

```diff
@@ -77,21 +76,21 @@ header {
     display: flex;
     justify-content: space-between;
 
-    padding-left: functions.pxToRem(10);
-    padding-right: functions.pxToRem(10);
-    height: functions.pxToRem(48);
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
```

Comment:
these previous `10px`s were illegal. Please let go and don't comment why this is `0.5rem`

Reply:
I don't have any vested interest in any of the paddings or margins, and I don't see who else would have, so don't worry 🤗

Reply:
Ah! Now I understand this comment. You're anticipating a comment about not changing the values in the refactor-commit… you know me too well 😂

Ok, tell you what, I'm not going to say it's ok, but I'll pretend I didn't see it… deal? 😊

Reply:
yes. exactly! 

perfect then 🤣